persistence/data_models_enums.go

```diff
@@ -78,19 +78,22 @@ func (e StateExecutionStatus) String() string {
 	}
 }
 
-type WorkerTaskType int32
+type ImmediateTaskType int32
 
 const (
-	WorkerTaskTypeWaitUntil WorkerTaskType = 1
-	WorkerTaskTypeExecute   WorkerTaskType = 2
+	ImmediateTaskTypeWaitUntil            ImmediateTaskType = 1
+	ImmediateTaskTypeExecute              ImmediateTaskType = 2
+	ImmediateTaskTypeNewLocalQueueMessage ImmediateTaskType = 3
```

Comment:
Probably plural to be more accurate : `ImmediateTaskTypeNewLocalQueueMessages`

Reply:
True.

Reply:
Wait, I think one row in local_queue table corresponds to one row in task table. So when client send 2 messages to local queue, we will insert two rows into local_queue table and insert two rows into task table.

If this is correct understanding, then the task type should be in a singular form?

Reply:
I will leave it in a singular form later. If we are going to store multiple messages in a row, we can rename it to the plural form.